lib/screens/explore_screens/regions_screens/edit_region_event/edit_region_event_image.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/components/flat_button_long.dart';
+import 'package:seeds/components/select_picture_box/select_picture_box.dart';
+import 'package:seeds/datasource/remote/model/firebase_models/region_event_model.dart';
+import 'package:seeds/design/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/domain-shared/event_bus/event_bus.dart';
+import 'package:seeds/domain-shared/event_bus/events.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+import 'package:seeds/screens/create_region_event_screens/interactor/viewmodels/create_region_events_page_commands.dart';
+import 'package:seeds/screens/explore_screens/regions_screens/edit_region_event/interactor/viewmodel/edit_region_event_bloc.dart';
+import 'package:seeds/utils/build_context_extension.dart';
+
+class EditRegionEventImage extends StatelessWidget {
+  const EditRegionEventImage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final event = ModalRoute.of(context)!.settings.arguments as RegionEventModel?;
+
+    return BlocProvider(
+      create: (_) => EditRegionEventBloc(event!),
+      child: BlocConsumer<EditRegionEventBloc, EditRegionEventState>(
+        listenWhen: (previous, current) => current.pageCommand != null,
+        listener: (context, state) {
+          if (state.pageCommand != null) {
+            final pageCommand = state.pageCommand;
+
+            if (pageCommand is ShowErrorMessage) {
+              eventBus.fire(ShowSnackBar(pageCommand.message));
+            }
+
+            //need  a page command for this screen
+            if (pageCommand is RemoveAuthenticationScreen) {
+              // This pop remove the authentication screen
+              Navigator.of(context).pop();
+            }
```

Comment:
That is due the image picker it  puts the app in on resume status right

Reply:
yep